common/helpers/digest.c

```diff
@@ -5,14 +5,13 @@
 #include "digest.h"
 
 #define TRYTE_LENGTH 2673
-#define TRANSACTION_LENGTH TRYTE_LENGTH * 3
 
 char* iota_digest(const char* trytes) {
   Curl curl;
   init_curl(&curl);
   curl.type = CURL_P_81;
 
-  size_t length = strnlen(trytes, TRANSACTION_LENGTH);
+  size_t length = strnlen(trytes, TRYTE_LENGTH);
```

Comment:
I think that was a mistake, can someone confirm ?

Reply:
guilty